docs/user/05-60-kyma-bindings.md

```diff
@@ -16,32 +31,87 @@ X-Broker-API-Version: 2.14
+KEB returns the `200 OK` status code with the kubeconfig in the response body. If the binding does not exist, the instance does not exist, or the instance is suspended, KEB returns a `404 Not Found` status code.
 
-## Unbinding
+All the codes are based on the [Open Service Broker API specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#fetching-a-service-binding) 
 
-To remove a binding, send a DELETE request to the Broker API:
+### Unbinding
+
+To remove a binding, send a DELETE request to KEB API:
 
 ```
 DELETE http://localhost:8080/oauth/v2/service_instances/{{instance_id}}/service_bindings/{{binding_id}}?plan_id={{plan_id}}&service_id={{service_id}}
 X-Broker-API-Version: 2.14
 ```
 
-If the binding is successfully removed, the Broker returns the `200 OK` status code. If the binding or service instance does not exist, the Broker returns the `410 Gone` code.
\ No newline at end of file
+If the binding is successfully removed, KEB returns the `200 OK` status code. If the binding or service instance does not exist, KEB returns the `410 Gone` code.
+
+All the codes are based on the [Open Service Broker API specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#unbinding) 
+
+## Bindings Management
+
+### Create Kyma Binding Process
+
+The binding creation process, that starts with a PUT HTTP request sent to `/oauth/v2/service_instances/{{instance_id}}/service_bindings/{{binding_id}}` endpoint, produces a binding with a kubeconfig that encapsulates JWT token used for user authentication. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such approach allows for modifying permissions granted with the kubeconfig.
+Besides the kubeconfig, the response contains metadata with the **expires_at** field, which specifies the expiration time of the kubeconfig. 
+To specify the duration for which the generated kubeconfig is valid explicitly, provide the **expiration_seconds** in the `parameter` object of the request body.
+
+
+The diagram below shows the flow of creating a Service Binding in Kyma Environment Broker. The process starts with a PUT request sent to KEB API. 
+
+![Bindings Create Flow](../assets/bindings-create-flow.drawio.svg)
+
+> **NOTE**: On the diagram error means forseen error in the process, not a server error.
+
+The creation process is devided into three parts: configuration check, request validation and binding creation.
+
+<!-- Configuration Check -->
+Given that a feature flag for Kyma Bindings is enabled, in the first instructions of the process KEB checks if the Kyma instance exists. If the instance is found and plan, that it has been provisioned with, is bindable, then KEB proceeds to validation phase.
```

Comment:
Is user interested in feature flag usage?

Reply:
Discussed.